FungibleToken.test.ts

```diff
@@ -34,44 +32,35 @@ const localChain = await Mina.LocalBlockchain({
 Mina.setActiveInstance(localChain)
 
 describe("token integration", () => {
-  let deployer: TestPublicKey
-  let sender: TestPublicKey
-  let receiver: TestPublicKey
-  let tokenAdmin: TestPublicKey
+  let deployer: Mina.TestPublicKey
+  let sender: Mina.TestPublicKey
+  let receiver: Mina.TestPublicKey
+  let tokenAdmin: Mina.TestPublicKey
   let tokenAdminContract: FungibleTokenAdmin
-  let newTokenAdmin: TestPublicKey
+  let newTokenAdmin: Mina.TestPublicKey
   let newTokenAdminContract: FungibleTokenAdmin
-  let tokenA: TestPublicKey
+  let tokenA: Mina.TestPublicKey
   let tokenAContract: FungibleToken
-  let tokenBAdmin: TestPublicKey
+  let tokenBAdmin: Mina.TestPublicKey
   let tokenBAdminContract: CustomTokenAdmin
-  let tokenB: TestPublicKey
+  let tokenB: Mina.TestPublicKey
   let tokenBContract: FungibleToken
-  let thirdPartyA: TestPublicKey
+  let thirdPartyA: Mina.TestPublicKey
   let thirdPartyAContract: ThirdParty
-  let thirdPartyB: TestPublicKey
+  let thirdPartyB: Mina.TestPublicKey
   let thirdPartyBContract: ThirdParty
 
   before(async () => {
     ;[deployer, sender, receiver] = localChain.testAccounts
+    ;[tokenAdmin, newTokenAdmin, tokenA, tokenBAdmin, tokenB, thirdPartyA, thirdPartyB] = Mina
+      .TestPublicKey.random(7)
```

Comment:
while you're cleaning up -- what bothers me about this test as well is that we unnecessarily split declaration and assignment of variables. why not create test keys at the top level of the file or describe block, instead of in `before()`? This would save us all the `let account: Mina.TestPublicKey` lines

Reply:
@kantp thoughts?

Reply:
Yes, I think that would be a good simplification.